index.html

```diff
@@ -2,24 +2,70 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="initial-scale=1, width=device-width,
-      shrink-to-fit=no">
-    <title>My Portfolio Website</title>
-    <link rel="stylesheet" href="./css/style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width" , initial-scale="1.0">
+    <title>My Portfolio website</title>
+
+    <!-- link external stylesheet -->
+    <link href="css/style.css" rel="stylesheet">
+
+    <!-- link fontawesome stylesheet -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+
+    <!-- link google fonts stylesheet -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Mulish:wght@200&display=swap"
+      rel="stylesheet">
+    <script src="https://kit.fontawesome.com/70cd6cd3b0.js"
+      crossorigin="anonymous"></script>
   </head>
 
   <body>
-    <!-- header section -->
-    <section class="header">
+    <!-- first section (header) -->
+    <section>
+      <header>
+        <div><a href="#">Mohamed Eldimardash</a></div>
+        <nav class="menu">
+          <a href="#"><img src="images/burger.png" alt="burger-menu"></a>
+        </nav>
+      </header>
+    </section> <!-- end section 1 -->
+
+    <!-- second section (headline) -->
+    <section>
+      <main class="headline">
```

Comment:
- [ ] As per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), please kindly align the content of this section with flexbox